src/diffusers/models/attention.py

```diff
@@ -194,10 +195,18 @@ def __init__(
         attention_bias: bool = False,
         only_cross_attention: bool = False,
         upcast_attention: bool = False,
+        norm_elementwise_affine: bool = True,
+        use_ada_layer_norm_zero: bool = False,
```

Comment:
Instead of making this a boolean, could we make it a string to be more general for the future? 
```suggestion
        norm_type: str = "layer", # ada_norm, ada_norm_zero
```

Reply:
Agree here's, let's make it a string.